README.md

```diff
@@ -5,44 +5,48 @@ The GOV.UK Pay Admin Users Module in Java (Dropwizard)
+| `DB_HOST`                                                                     | The hostname of the database server. |
+| `DB_NAME`                                                                     | The name of the database on `DB_HOST`. Defaults to `adminusers`. |
+| `DB_PASSWORD`                                                                 | The password for the `DB_USER` user. |
+| `DB_PORT`                                                                     | The port number it use when connecting to the database server. Defaults to `5432`. |
+| `DB_SSL_OPTION`                                                               | To turn TLS on this value must be set as `ssl=true`. Otherwise must be empty. |
+| `DB_USER`                                                                     | The username to log into the database as. |
+| `FORGOTTEN_PASSWORD_EXPIRY_MINUTES`                                           | The number of minutes password reset tokens are valid for. Defaults to `90`. |
+| `JAVA_HOME`                                                                   | The location of the JRE. Set to `/opt/java/openjdk` in the `Dockerfile`. |
+| `JAVA_OPTS`                                                                   | Commandline arguments to pass to the java runtime. Optional. |
+| `JPA_LOG_LEVEL`                                                               | The logging level to set for JPA. Defaults to `WARNING`. |
+| `JPA_SQL_LOG_LEVEL`                                                           | The logging level to set for JPA SQL logging. Defaults to `WARNING`. |
+| `LOGIN_ATTEMPT_CAP`                                                           | The number of consecutive failed logins a user can have before their account is disabled. Defaults to `10`. |
+| `METRICS_HOST`                                                                | The hostname to send graphite metrics to. Defaults to `localhost`. |
+| `METRICS_PORT`                                                                | The port number to send graphite metrics to. Defaults to `8092`. |
+| `NOTIFY_2FA_TEMPLATE_ID`                                                      | The GOV.UK Notify template ID to use for sending MFA codes via SMS. Defaults to `pay-notify-two-factor-template-id`. |
+| `NOTIFY_SIGN_IN_OTP_SMS_TEMPLATE_ID`                                          | The GOV.UK Notify template ID to use for sending OTP codes via SMS for changing the sign-in method to text messages. Defaults to `pay-notify-sign-in-otp-sms-template-id`. |
+| `NOTIFY_CHANGE_SIGN_IN_2FA_TO_SMS_OTP_SMS_TEMPLATE_ID`                        | The GOV.UK Notify template ID to use for sending OTP codes via SMS for changing the sign-in method to text messages. Defaults to `pay-notify-switch-sign-in-2fa-to-sms-otp-sms-template-id`. |
```

Comment:
The description of `NOTIFY_CHANGE_SIGN_IN_2FA_TO_SMS_OTP_SMS_TEMPLATE_ID` and `NOTIFY_SIGN_IN_OTP_SMS_TEMPLATE_ID` are the same: `The GOV.UK Notify template ID to use for sending OTP codes via SMS for changing the sign-in method to text messages.` Is this correct?

Reply:
Nope, that’s a mistake! I’ll fix that.